lnd_test.go

```diff
@@ -5514,17 +5514,8 @@ func testGraphTopologyNotifications(net *lntest.NetworkHarness, t *harnessTest)
 func testNodeAnnouncement(net *lntest.NetworkHarness, t *harnessTest) {
 	ctxb := context.Background()
 
-	ipAddresses := map[string]bool{
-		"192.168.1.1:8333":                            true,
-		"[2001:db8:85a3:8d3:1319:8a2e:370:7348]:8337": true,
-	}
-
-	var lndArgs []string
-	for address := range ipAddresses {
-		lndArgs = append(lndArgs, "--externalip="+address)
-	}
-
-	dave, err := net.NewNode("Dave", lndArgs)
+	advertisedAddr := "192.168.1.1:8333"
```

Comment:
What about the IPv6 address?

Reply:
Eventually this should also be modified to expect that Tor addresses are properly advertised as well.

Reply:
Fixed.